### ISSUE TITLE
Training works perfectly on example datasets, fails on custom datasets

### ISSUE THREAD

**Issue by Tylersuard** [chrisdonahue/wavegan#85]
Hello.  I was able to train on the Piano dataset successfully.  I downloaded a file from that dataset and found it to be a 32bit 48k wav file.  I converted my own mp3 dataset to the same format and attempted to train on it... no success.  Received the "scipy cannot change sample rate" error shown in another issue.  Tried switching to Librosa by removing --data_fast_wav flag, caused more errors than it solved.

**Comment by xyz010** [comment]
Do you mean 48kHz sampling rate? It appears to be too high, and it will take forever to train.

**Comment by Tylersuard** [comment]
Yes, thank you for your comment, I meant 48khz sampling rate.  The files used in the example Piano dataset were 24bit 48khz files.  When I converted my own .wav files to that format and attempted to train on them, I received the error:

File "/tensorflow-1.15.2/python3.6/tensorflow_core/python/ops/script_ops.py", line 235, in __call__
    ret = func(*args)

  File "/content/wavegan/loader.py", line 128, in <lambda>
    fast_wav=decode_fast_wav)

  File "/content/wavegan/loader.py", line 23, in decode_audio
    _fs, _wav = wavread(fp)

  File "/usr/local/lib/python3.6/dist-packages/scipy/io/wavfile.py", line 257, in read
    "has {}-bit data.".format(bit_depth))

ValueError: Unsupported bit depth: the wav file has 24-bit data.
